### PR TITLE
Bug fixes for 2.1.1

### PR DESCRIPTION
**Pull request by chelm**
* Fixing a filtering bug with duplicate VRT files names 
* Adding some docs to the index file 
* fixed the idFilter clause for non-worker exports

### CODE REVIEW

**Comment by chelm** [comment]
dont merge this just yet may have found something I need to fix

**Comment by chelm** [comment]
nevermind all good

**Comment by chelm** [comment]
@ngoldman beep

File: lib/Exporter.js [hunk 1]
```diff
@@ -97,10 +97,9 @@ exports.exportLarge = function( koop, format, id, key, type, options, finish, do
   var q = async.queue(function (task, cb) {
     // instead of passing a limit and offset 
     // we use a WHERE clause 
-    var idFilter = ' id >= '+ options.offset + ' AND id < ' + options.offset + options.limit;
-    var idFilter = ' id >= '+ task.offset + ' AND id < ' + (parseInt(task.offset) + parseInt(task.options.limit));
+    var idFilter = ' id >= '+ options.offset + ' AND id < ' + (parseInt(options.offset) + parseInt(options.limit));
     var opts = {
-      ifFilter: idFilter,
+      idFilter: idFilter,
```
Comment (ngoldman) [inline lib/Exporter.js]: hehe

File: index.js [hunk 2]
```diff
@@ -87,6 +90,12 @@ module.exports = function( config ) {
     }
   };
 
+  /** 
+   * Registers a koop plugin
+   * Plugins can be any function that you want to have global access to 
+   * within koop provider models  
+   * @param {object} any koop plugin
+   */
```
Comment (ngoldman) [inline index.js]: this is great, more internal method doc is always good when possible

**Comment by ngoldman** [comment]
@chelm :+1: lgtm

**Pull request merged**